python/pynessie/_log.py

```diff
@@ -18,6 +16,7 @@ def show_log(
     number: Optional[int] = -1,
     after: Optional[str] = None,
     before: Optional[str] = None,
+    author: Optional[str] = None,
```

Comment:
Wonder whether we should remove the filtering parameter and use "Python kwargs" for the filtering parameters.

Reply:
+1

Reply:
fixed